tests/js/spec/components/modals/dashboardWidgetLibraryModal.spec.jsx

```diff
@@ -117,4 +119,20 @@ describe('Modals -> DashboardWidgetLibraryModal', function () {
 
     container.unmount();
   });
+
+  it('disables save button if widget limit is exceeded', function () {
+    // Checking initial modal states
+    const mockApply = jest.fn();
+    const closeModal = jest.fn();
+    types.MAX_WIDGETS = 1;
+    const container = mountModal({initialData}, mockApply, closeModal);
+
+    const selectButtons = screen.getAllByRole('button');
+    userEvent.click(selectButtons[3]);
+    userEvent.click(selectButtons[4]);
+
+    expect(screen.getByTestId('confirm-widgets')).toBeDisabled();
+
+    container.unmount();
```

Comment:
Can this be in `afterEach` too? :eyes: I kind of though RTL automatically unmounted test components though. I could be wrong, limited use but it sounded familiar

Reply:
good catch!

Reply:
86b8ec5beaf78db0e3ceda9c2774d4c5a7e00632